concurrent/actor/Actor.java

```diff
@@ -88,8 +72,25 @@ public void tell(Consumer<STATE> job) {
         return future;
     }
 
-    public EventLoop.ScheduledJob schedule(long millis, Consumer<STATE> job) {
+    public EventLoop.ScheduledJob schedule(long frequencyMs, Consumer<STATE> job) {
```

Comment:
This is not `frequencyMs`. This is the absolute time that the job will run. E.g. `eventLoop.schedule(System.currenetTime() + 100, () -> {})`.